guides/wide-ep-lws/README.md

```diff
@@ -197,3 +197,7 @@ kubectl delete -k ./manifests/gateway/<gke-l7-regional-external-managed|istio|kg
 ## Customization
 
 For information on customizing a guide and tips to build your own, see [our docs](../../docs/customizing-a-guide.md)
+
+## Topology Aware Scheduling (TAS)
+
+For information on how to use topology aware scheduling using Kueue, see [LWS + TAS user guide](https://lws.sigs.k8s.io/docs/examples/tas/). The GKE example already has the required labels.
```

Comment:
I want to have more detail in the guide -the core gap is that there is no gang scheduling and affinity scheduling in Kube is best effort or high toil.  So I think up near the top we need to draw out both of those as sproblems, and present Kueue + TAS today (and grove and then later core kube gang scheduling) as solutions.

A guide is roughly intended to be:

1. Optimization opportunity
2. Tradeoffs / considerations
3. Prereqs to try / choices you have to make
4. Setup
5. Validation / Benchmarking

I think you either want to add those to 2 if they are fundamental to wide ep's tradeoffs, or 3 if tehy are a requiremetns to use wide ep well.